src/rpc/mining.cpp

```diff
@@ -454,9 +454,9 @@ UniValue getblocktemplate(const UniValue& params, bool fHelp)
             "  \"finalsaplingroothash\" : \"xxxx\",   (string) (DEPRECATED) The hash of the light client root field in the block header\n"
             "  \"defaultroots\" : {                 (json object) root hashes that need to be recomputed if the transaction set is modified\n"
             "     \"merkleroot\" : \"xxxx\"           (string) The hash of the transactions in the block header\n"
-            "     \"authdataroot\" : \"xxxx\"         (string) The hash of the authorizing data merkel tree\n"
             "     \"chainhistoryroot\" : \"xxxx\"     (string) The hash of the chain history\n"
-            "     \"blockcommitmentshash\" : \"xxxx\" (string) The hash of the block commitments field in the block header\n"
+            "     \"authdataroot\" : \"xxxx\"         (string) (From NU5) The hash of the authorizing data merkel tree\n"
```

Comment:
```suggestion
            "     \"authdataroot\" : \"xxxx\"         (string) (From NU5) The hash of the authorizing data Merkle tree\n"
```

Reply:
We can fix this pre-existing typo in a follow-up PR post-hotfix.